src/main/java/com/kodilla/ecommercee/ProductNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.ecommercee;
+
+public class ProductNotFoundException extends Exception {
```

Comment:
Dla zdarzeń "biznesowych" raczej to powinien być wyjątek RuntimeException, chyba że założyliście w projekcie że używacie Exception

Reply:
Zmienione